### ISSUE TITLE
Refs broken for inputs with :value

### ISSUE THREAD

**Issue by martinklepsch** [tonsky/rum#212]
```clj
(rum/defcs InputRefTestHook
  [s]
  (let [r (rum/create-ref)]
  [:input
   {:width 20
    :ref r
    :value "test" 
    :on-focus #(js/console.log (rum/deref r))}]))
```
log output with `:value "test"`

<img width="684" alt="CleanShot 2020-06-12 at 10 03 22@2x" src="https://user-images.githubusercontent.com/97496/84479932-065b3780-ac94-11ea-8a4b-f0311ed2b2d8.png">

log output without `:value`

<img width="187" alt="CleanShot 2020-06-12 at 10 03 43@2x" src="https://user-images.githubusercontent.com/97496/84479944-0b1feb80-ac94-11ea-912f-5164336316d4.png">

The old approach of using `rum/dom-node` seems to work fine with inputs that have a `:value` key.

**Comment by roman01la** [comment]
Hmmmm, I _think_ this might be related to the fact that Sablono wraps inputs into a component (to work around async updates and jumping caret issues).

So with `value` set you get an instance of a wrapper component, but w/o `value` attr Sablono doesn't wrap the input which gives you a plain DOM node.

**Comment by roman01la** [comment]
Once https://github.com/tonsky/rum/pull/205 goes in we won't have input wrappers anymore.

**Comment by g7s** [comment]
Yeah this issue has been unresolved since 2016 https://github.com/r0man/sablono/issues/121

**Comment by roman01la** [comment]
fixed in 0.12.0
